packages/renderer/src/lib/objects/KubernetesObjectsList.svelte

```diff
@@ -45,6 +47,9 @@ let resources = $state<{ [key: string]: KubernetesObject[] | undefined }>({});
 let resourceListeners: (IDisposable | undefined)[] = [];
 let legacyUnsubscribers: Unsubscriber[] = [];
 
+let kubeActionsDisabled: boolean = $state(false);
+let selectedResource: ResourceName = $state('deployments');
```

Comment:
Should not have deployment-specific code in this file or default to deployments.